src/libraries/System.Runtime.Serialization.BinaryFormat/src/System/Runtime/Serialization/BinaryFormat/RecordType.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.Serialization.BinaryFormat;
+
+/// <summary>
+/// Record type.
+/// </summary>
+/// <remarks>
+///  <para>
+///   The enumeration does not contain all values supported by the <see href="https://learn.microsoft.com/openspecs/windows_protocols/ms-nrbf/954a0657-b901-4813-9398-4ec732fe8b32">
+///   [MS-NRBF] 2.1.2.1</see>, but only those supported by the <see cref="PayloadReader"/>.
+///  </para>
+/// </remarks>
+#if SYSTEM_RUNTIME_SERIALIZATION_BINARYFORMAT
+public
+#else
+internal
+#endif
+enum RecordType : byte
+{
+    /// <summary>
+    /// The NRBF header (the first record in NRBF payload).
+    /// </summary>
+    SerializedStreamHeader = 0,
+    /// <summary>
+    /// A class information that references another class record's metadata.
+    /// </summary>
+    ClassWithId = 1,
+    // SystemClassWithMembers and ClassWithMembers are not supported by design (require type loading)
+    /// <summary>
```

Comment:
The comment that says you're skipping 2 and 3 is buried now.  I recommend adding some vertical whitespace to this file.

```suggestion
    ClassWithId = 1,
    
    // SystemClassWithMembers and ClassWithMembers are not supported by design (require type loading)
    
    /// <summary>
```

and generally repeating to have a blank line between the declaration of the enum member and the start of the successive member's doc comment.